plugins/talk-plugin-auth/client/components/ForgotContent.js

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import styles from './styles.css';
-import {Button} from 'plugin-api/beta/client/components/ui';
+import {Button, TextField} from 'plugin-api/beta/client/components/ui';
 import t from 'coral-framework/services/i18n';
 
 class ForgotContent extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      value: '',
+    };
+  }
```

Comment:
Instead of writing a full _constructor_ we can simply rely on property initialisers in this case:

```js
class ForgotContent extends React.Component {

  state = {value: ''};
 
  handleSubmit = (e) => {
    ...
```